src/Compute/v2/Service.php

```diff
@@ -0,0 +1,15 @@
+<?php
+
+namespace OpenStack\Compute\v2;
+
+use OpenStack\Common\Service\AbstractService;
+
+class Service extends AbstractService
+{
+    public function createServer($name, $imageId, $flavorId, array $options = [])
```

Comment:
Are you following an idiom of have required parameters be positional and the last positional parameters be an array for all optional parameters?